app/javascript/components/upload/uploadUtils.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
```

Comment:
This file name should use either UpperCamelCase or hyphen-case, like all other SCP JS files.

The `Utils` suffix indicates it should be in `javascript/lib` and not `javascript/components`, but I'd change the name instead of the location given this module is all components.  

Something like `UploadWidgets.js` or `UploadComponents.js` would clarify at a glance of the file name that it's React components related to upload, and not a bunch of framework-agnostic JavaScript functions.